code/modules/necromorph/corruption/growth.dm

```diff
@@ -82,8 +82,8 @@
 	density = FALSE
 	marker_spawnable = FALSE
 
-	range = 40
-	speed = 3
+	range = 12
+	speed = 2
```

Comment:
this is a crazy degree of change to do all at once, you have no idea how this is going to affect things. 
Go with a smaller change, like to 30 to start with and see how it goes

Reply:
Nanako, the main issue is that roots support UP TO 40 RANGE OF TILES. 
The way this looks in practice? You place a root, everywhere within 40 tiles starts to spread at an incredible fast rate. Humans delete a root? it keeps going, because there's likely a root somewhere within 40 tiles that keeps the corruption there. This is absolutely needed. It doesn't even spread well in maint, because it might be expanding off the other end of the ship, because again, 40 tile range. it's instead being used as the only root, because it just spreads faster with the slight negative of requiring a few more nodes than branches.